src/en/news/blog/2022/scaletesting-with-pawsey/index.md

```diff
@@ -0,0 +1,82 @@
+
+### Core
+
+- All Ceph daemons report their performance counters to the active manager every 5 seconds by default. We found that the responsiveness of the mgr and CLI became intermittent beyond 2,800 OSDs due to this data load and the interaction with the mgr/prometheus module. A [PR](https://github.com/ceph/ceph/pull/44162) was merged which reduced this issue significantly.
+- A bug was identified in the scale up profile of the [`pg_autoscaler`](https://docs.ceph.com/en/quincy/rados/operations/placement-groups/#autoscaling-placement-groups), which prevented pg’s from peering. This has been subsequently fixed, and backported to [Pacific](https://github.com/ceph/ceph/pull/44869) and [Octopus](https://github.com/ceph/ceph/pull/44541).
+
+### Orchestrator/cephadm
+
+- `ceph-volume` was enhanced to better handle unexpected disk formats like Atari partitions and GPT based devices
+- `cephadm` uses ssh as the control path to each cluster host. During our negative testing, we found that when ssh issues were encountered (bad/missing keys), the resulting information returned to the user was overly verbose. This was fixed in [PR 43880](https://github.com/ceph/ceph/pull/43880).
+- Cephadm’s experimental 'agent' feature was used to provide the orchestrator with host, device and daemon state metadata. Testing showed that deployment and data quality were as expected, but the agent generated too much log traffic, making potential troubleshooting problematic. The intent is for the Agent to become a supported feature in the Ceph Reef release.
+- With so many hosts, the lack of filtering in the `orch host ls command` made interacting with the cluster more difficult. Filtering was added in [PR 44020](https://github.com/ceph/ceph/pull/44020), addressing this usability issue.
+
+### UI Management
+
+Historically, Ceph clusters of this size have been managed with a CLI-based toolchain, but we took this opportunity to briefly look at the integrated Ceph UI (dashboard).
+
+Whilst the monitoring capabilities of the dashboard were functional, management of hosts and OSDs was problematic leading to browser timeouts.
+
+The lack of pagination has been identified as the root cause for most of these responsiveness issues. Work is currently under way to introduce pagination to the dashboard's backend services layer ([PR 46644](https://github.com/ceph/ceph/pull/46644/)).
+
+### Monitoring
+
+Out-of-the-box, `cephadm` deploys a Prometheus server to provide the cluster with monitoring and alerting capabilities. The data sent to Prometheus is emitted by the mgr/prometheus module which interacts with ceph-mgr to present Ceph counters in Prometheus exposition format.
+
+As the cluster scaled beyond 2,800 OSDs the Prometheus server was unable to consistently retrieve data from the mgr/prometheus endpoint, primarily due to the volume of data returned (> 400K samples from a single endpoint).
```

Comment:
While there were other reasons for that too, maybe worth mentioning the new distributed Ceph metrics exporter effort (it should also improve metrics scaling)?

Reply:
there is another blog planned to cover the internal scale testing I did in the scalelab (jan/feb 2022) - that's where a discussion about the exporter strategy will land. OK?